### DISCUSSION TITLE
Idea: rename "prompt_weight_x" variables

### DISCUSSION THREAD

**Discussion by sashaagafonoff** [rewbs/sd-parseq#104] [General]
Being a distractable sort, I often forget which prompt weight variable affects what... 

It would be super neat to be able to redefine the "prompt_weight_x" variables with a custom name so that those will appear in the keyframes grid heading in place of the current generic names:

eg prompt contains the following text:

`<hypernet:TID17500:${hypernetwork_weight:[value]} >`

or 

`(futuristic city, surrealist, propaganda, (an 80s sci-fi film), (by junji ito and jerome opena, full colour), photograph, HD, HDR, 4K, 35mm, 1970s science fiction films, noire, retro futurism, aliens, psychedelic prismatic gradient (3D) vector geometry:${aesthetic_prompts_weight})`

**Comment by rewbs** [comment]
Good idea. The way I'm planning to implement this is, instead of allowing renaming or aliases on existing parameters, you'd be able to define entirely custom parameters which you can then reference from prompts, (and potentially from formulas for other params if there's a clear evaluation ordering). What do you think?

**Reply by sashaagafonoff** [reply]
yes, even better!!! intriguing idea to be able to reference them from formulas - i've definitely wanted that a few times.

...do you think this is how new operating systems are born? :)

**Reply by rewbs** [reply]
Cool, this idea is tracked here: https://github.com/rewbs/sd-parseq/issues/36
